quam_builder/architecture/quantum_dots/README.md

```diff
@@ -0,0 +1,682 @@
+# GateSet & VirtualGateSet: Orchestrating DC Voltage Control in QUA & Abstracting Gate Control with Virtualisation Layers
```

Comment:
That's very nice, and while it is true that for now we only have the gateset and virtual_gateset components, we might need to rename it when we'll have the full QPU abstraction for spin qubits in here. That being said, I am good with keeping it as is for now and change it later.